src/credentials/GroupDataProvider.h

```diff
@@ -81,29 +81,37 @@ class GroupDataProvider
     struct GroupKey
     {
         GroupKey() = default;
-        GroupKey(chip::GroupId group, chip::KeysetId keyset) : group_id(group), keyset_id(keyset) {}
+        GroupKey(GroupId group, KeysetId keyset) : group_id(group), keyset_id(keyset) {}
         // Identifies group within the scope of the given Fabric
-        chip::GroupId group_id = kUndefinedGroupId;
+        GroupId group_id = kUndefinedGroupId;
         // Set of group keys that generate operational group keys for use with this group
-        chip::KeysetId keyset_id = 0;
+        KeysetId keyset_id = 0;
         bool operator==(const GroupKey & other) { return this->group_id == other.group_id && this->keyset_id == other.keyset_id; }
     };
 
     struct GroupEndpoint
     {
         GroupEndpoint() = default;
-        GroupEndpoint(chip::GroupId group, chip::EndpointId endpoint) : group_id(group), endpoint_id(endpoint) {}
+        GroupEndpoint(GroupId group, EndpointId endpoint) : group_id(group), endpoint_id(endpoint) {}
         // Identifies group within the scope of the given Fabric
-        chip::GroupId group_id = kUndefinedGroupId;
+        GroupId group_id = kUndefinedGroupId;
         // Endpoint on the Node to which messages to this group may be forwarded
-        chip::EndpointId endpoint_id = kInvalidEndpointId;
+        EndpointId endpoint_id = kInvalidEndpointId;
 
         bool operator==(const GroupEndpoint & other)
         {
             return this->group_id == other.group_id && this->endpoint_id == other.endpoint_id;
         }
     };
 
+    struct SessionKey
+    {
+        SessionKey()     = default;
+        GroupId group_id = kUndefinedGroupId;
+        FabricIndex fabric_index;
+        Crypto::SecureKeyContext * key = nullptr;
```

Comment:
Suggest `SymmetricKeyContext`, which is more "talking" than `SecureKeyContext`

Reply:
Works for me. Renamed as SymmetricKeyContext.